ota-plus-web/docs/modules/ROOT/nav.adoc

```diff
@@ -2,11 +2,10 @@
 .Get started
 * xref:index.adoc[Introduction] 
 * xref:get-access.adoc[Get access]
-// TODO * xref:switch-orgs.adoc[Switching Organizations]
 * xref:ota-workflow-campaign-manager.adoc[OTA update workflow]
 
 .Devices
-* xref:manage-device-prov.adoc[Provisioning]
+* xref:manage-device-prov.adoc[Whta is device provisioning?]
```

Comment:
Typo here.  I think you meant `What` instead.

Reply:
Yep, thank you!
Corrected